front/components/authPage/AuthPage.vue

```diff
@@ -13,6 +13,9 @@
       />
     </v-col>
     <v-col cols="12" md="4" class="auth-page__content">
+      <v-icon class="auth-page__go-back-icon hidden-md-and-up" @click="goBack()"
```

Comment:
for consistency you dont really need to add () just @click="goBack"

Reply:
Okay, I will leave it out going forward.